.gitignore

```diff
@@ -112,6 +112,7 @@ celerybeat.pid
 .venv
 env/
 venv/
+venv3.7/
```

Comment:
Can you revert this one? Not a big deal, but I'd prefer to keep `.gitignore` out of this PR

Reply:
I'm deeply sorry, this one was my internal testing change. I must have mistakenly added it. :cry: